modules/shapefile/test/shp/parse-shp.spec.js

```diff
@@ -0,0 +1,65 @@
+import test from 'tape-promise/tape';
+import parseShape from '@loaders.gl/shapefile/lib/parse-shp';
+import {fetchFile} from '@loaders.gl/core';
+import {geojsonToBinary} from '@loaders.gl/gis';
+
+const BOSTOCK_DATA_FOLDER = '@loaders.gl/shapefile/test/data/shapefile-js';
```

Comment:
Remember the license, the man does not want his name to be spoken in vain.

Reply:
> The name Michael Bostock may not be used to endorse or promote products derived from this software without specific prior written permission.

I didn't read that as "I can't reference his name in code"; I read it as I can't say "according to Michael Bostock X product is great" 

Regardless I can replace with `SHAPEFILE_JS`